### DISCUSSION TITLE
Integration of HTML in releases notes

### DISCUSSION THREAD

**Discussion by A-d-r-i** [chocolatey/home#88] [Ideas]
Hello,
Would it be possible to add this function (reading HTML) on the site (or in Markdown, which, if I'm not mistaken, integrates basic HTML)?
This would be very useful in cases where we automatically extract the release notes from the official site (ex: [Mendeley-Reference-Manager](https://community.chocolatey.org/packages/mendeley-reference-manager/2.61.1#releasenotes)).

Thanks for your informations!

**Comment by vexx32** [comment]
I thought release notes / description fields already incorporated Markdown support? /cc @designxsteph

**Comment by A-d-r-i** [comment]
Absolutely, it incorporates Markdown without problem (I use it happily) but the HTML is not formatted in my example.

**Comment by pauby** [comment]
I've converted this over to a discussion.

**Reply by A-d-r-i** [reply]
Some news regarding this issue?

**Comment by pauby** [comment]
This isn't something we have any plans to implement.

The information in the `.nuspec` file should be markdown. If you need to display HTML then my suggestion would be to provide a link to the release notes rather than embedding them.

**Reply by A-d-r-i** [reply]
Ok, thanks @pauby 😊

**Reply by gep13** [reply]
Just to follow up on this...

Allowing direct entry of HTML, and rendering of that HTML on the site, would be an avenue for attack, as a malicious user could put HTML/JavaScript/CSS in there, which would then be output to the package page.  This is not something that we are going to be doing.  We will only be supporting markdown within the nuspec file.

**Reply by A-d-r-i** [reply]
Thanks for the details Gary! I understand perfectly. I'm going to start a conversion from html to markdown
